lib/tests/streamlit/runtime/media_file_manager_test.py

```diff
@@ -325,51 +331,44 @@ def test_remove_orphaned_files_multiple_sessions(self, mock_get_session_id):
                     sample["content"], sample["mimetype"], coord
                 )
 
-        self.assertEqual(len(self.media_file_manager), len(VIDEO_FIXTURES))
+        self.assertEqual(
+            len(self.media_file_manager._file_metadata), len(VIDEO_FIXTURES)
+        )
+
+        file_ids = list(self.media_file_manager._file_metadata.keys())
 
         # Remove session1's references
         mock_get_session_id.return_value = "mock_session_1"
         self.media_file_manager.clear_session_refs()
         self.media_file_manager.remove_orphaned_files()
 
         # The files are all still referenced by session_2
-        self.assertEqual(len(self.media_file_manager), len(VIDEO_FIXTURES))
+        self.assertEqual(
+            len(self.media_file_manager._file_metadata), len(VIDEO_FIXTURES)
+        )
+
+        # MediaFileStorage.delete_file should not have been called yet...
+        storage_delete_spy.assert_not_called()
 
         # Remove session2's references, but don't call "remove_orphaned_files" yet...
         mock_get_session_id.return_value = "mock_session_2"
         self.media_file_manager.clear_session_refs()
 
         # The files still exist, because they've only been de-referenced and not
         # removed.
-        self.assertEqual(len(self.media_file_manager), len(VIDEO_FIXTURES))
+        self.assertEqual(
+            len(self.media_file_manager._file_metadata), len(VIDEO_FIXTURES)
+        )
+
+        # MediaFileStorage.delete_file should not have been called yet...
+        storage_delete_spy.assert_not_called()
 
         # After a final call to remove_orphaned_files, the files should be gone.
         self.media_file_manager.remove_orphaned_files()
-        self.assertEqual(len(self.media_file_manager), 0)
-
-    def test_media_file_url(self):
-        self.assertEqual(MediaFile("abcd", b"", "audio/wav").url, "/media/abcd.wav")
-        self.assertEqual(MediaFile("abcd", b"", "image/jpeg").url, "/media/abcd.jpeg")
-        self.assertEqual(MediaFile("abcd", b"", "video/mp4").url, "/media/abcd.mp4")
-        self.assertEqual(MediaFile("abcd", b"", "video/webm").url, "/media/abcd.webm")
```

Comment:
This is testing MediaFileStorage functionality, now. (We have an equivalent test in `memory_media_file_storage_test.py`)